docker-compose.yml

```
@@ -9,9 +9,9 @@ services:
       POSTGRES_PASSWORD: ${POSTGRES_PASSWORD:-password}
       PGDATA: /data/postgres
     volumes:
-      - postgres:/data/postgres
+      - postgres:/var/lib/postgresql/data
     ports:
-      - "5432:5432"
+      - "5555:5432"
```

Pourquoi ? 🤔 
A la limite ajouter une variable d'env, si t'as besoin de le changer de ton côté
```suggestion
      - "${POSTGRES_PORT:-5432}:5432"
```

---

Ok